Add computed Baseline status

rviscomi opened the pull request:
Progress on https://github.com/microsoft/vscode-css-languageservice/issues/427

- ran `npm run update-sources`
- added `compute-baseline` dependency
- added Baseline status for each feature
- merged Baseline-supported browsers with the `browsers` field
- ran `npm run generate-data`

I think this also fixes a bug where some pseudo-elements and pseudo-classes might have `()` suffixes that need to get stripped out to match the feature IDs in BCD.

See PR https://github.com/microsoft/vscode-css-languageservice/pull/428 for the corresponding hovercard UI changes

aeschli commented:
We don't want to make any breaking changes to the custom data format, just additions. 
The file is spec'ed and also used by other projects. So we want to keep the `browsers` as is.

So simply add a new 'baseline' (or `baseline-compatibility') property on the same level as browsers. The value set should be well defined, maybe an enum and a description. This can then go in the schema: https://github.com/microsoft/vscode-CSS-languageservice/blob/main/docs/customData.schema.json and also in the documentation.

aeschli commented:
https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility

rviscomi commented:
Thanks @aeschli. I've reverted the `browsers` change and added `baseline` to [customData.schema.json](https://github.com/microsoft/vscode-css-languageservice/pull/428/files#diff-ce3183913c02a9c437b6a75c50626297d5d49c6c9911224f3348b5fe7c7a315c). Is there a schema validator I can run to make sure it's correct?

aeschli commented:
If you open it in in VS Code you will get validation.
To validate an custom data file against your schema follow the steps here: https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings

rviscomi commented:
Ok great, I've added this to my local settings and I can confirm that there are no validation warnings in `browsers.css-data.json`:

```json
    "json.schemas": [
        {
            "fileMatch": [
                "**/web-data/data/browsers.css-data.json"
            ],
            "url": "/Users/rviscomi/git/vscode-css-languageservice/docs/customData.schema.json"
        }
    ]
```

aeschli commented on web-data/css/mdn/mdn-browser-compat-data-importer.js:
```diff
@@ -82,8 +90,11 @@ function addMDNReferences(atdirectives, pseudoclasses, pseudoelements, propertie
 const browserNames = {
 	E: 'Edge',
 	FF: 'Firefox',
+  FM: 'Firefox_Android',
```
I would use `FFA` and `CA`. Who knows, one day Chrome is on IOS as well...

aeschli commented:
Thanks @rviscomi !
